relay-cabi/include/relay.h

```diff
@@ -579,4 +579,9 @@ struct RelayStr relay_validate_sampling_condition(const struct RelayStr *value);
  */
 struct RelayStr relay_validate_sampling_configuration(const struct RelayStr *value);
 
+/**
+ * Validate entire project config. If `strict` is true, checks for unknown fields in the input.
```

Comment:
"checks" in the explanation of `strict` is confusing. I find more clear the sentence in the relay function itself:

> If `strict` is true, verify that reserializing the instance results in the same fields.

If you want to have a not-so-defined sentence here to leave room for modifications on the other side, I'd suggest linking to the other function in `relay-dynamic-config/src/utils.rs` and removing the ambiguity.